src/components/model.js

```diff
@@ -90,4 +99,7 @@ module.exports.Component = registerComponent('loader', {
       el.setObject3D('mesh', collada.scene);
     });
   }
-});
+};
+
+module.exports.Component = registerComponent('loader', ModelComponent);
+module.exports.Component = registerComponent('model', ModelComponent);
```

Comment:
Okay, I think it was sensible to deprecate, but I changed it to your preference.